Makefile

```diff
@@ -133,13 +124,13 @@ test-list: test/test-list.js
 test/test-list.js: force
 	@git ls-files -o -c test/feature | node build/build-test-list.js > $@
 
-test-interpret: test/unit/runtime/test_interpret.js
+test-interpret: test/unit/runtime/traceur-interpreter.js
 	./traceur $^
 
-test-interpret-throw: test/unit/runtime/throwsError.js
+test-interpret-throw: test/unit/runtime/resources/throwsError.js
```

Comment:
`throwsError.js` is not a test.

I do think there is one file in `resources/` which calls `assert` but the goal of the test is to verify that this file is loaded and that the code it executes can call the System loader. So its both a resource and a test.